### ISSUE TITLE
I/O timeout on upgrade

### ISSUE THREAD

**Issue by ain** [docker/machine#1937]
Following the ideas from the issue of docker/docker#16746, running 

        docker-machine upgrade <machine>

fails with 

```
$ docker-machine upgrade dev
Stopping machine to do the upgrade...
Upgrading machine dev...
Get https://api.github.com/repos/boot2docker/boot2docker/releases: dial tcp: i/o timeout
```

The platform is OS X El Capitan with
```
$ docker version
Client:
 Version:      1.8.2
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   0a8c2e3
 Built:        Thu Sep 10 19:10:10 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      1.8.1
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   d12ea79
 Built:        Thu Aug 13 02:49:29 UTC 2015
 OS/Arch:      linux/amd64
```

**Comment by nathanleclaire** [comment]
I've got a fix for this in the pipe

**Comment by nathanleclaire** [comment]
Hm, actually, it seems I have mis-read this issue and I'm thinking of a different one.  Do subsequent `docker-machine upgrade` commands work, or do you get the same error?  How about restarting the VM, does it fix the issue?

**Comment by ain** [comment]
No to all questions, unfortunately. Tried multiple times. On a different machine that still runs 1.8.1, the error is slightly different:

        $ docker-machine upgrade dev
        Stopping machine to do the upgrade...
        exit status 1

**Comment by ain** [comment]
Problem resolved by swopping Google DNS so that `8.8.8.8` became primary and `4.4.4.4` secondary DNS server. May have been DNS cache issue.

**Comment by nathanleclaire** [comment]
Thanks for following up to close @ain
